crates/storage/src/libmdbx.rs

```diff
@@ -135,6 +122,12 @@ impl Debug for Store {
 }
 
 // Define tables
+
+table!(
+    /// Block hash to number table.
+    ( BlockNumbers ) BlockHashRLP => BlockNumber
```

Comment:
Is there a reason for not using the BlockHeaders table for this same functionality?
It includes more data, but still the BlockNumber is stored in it.

Reply:
Ah, you are right... missed that.